pyart/retrieve/__init__.py

```diff
@@ -27,4 +27,10 @@
 from .gate_id import map_profile_to_gates, fetch_radar_time_profile
 from .simple_moment_calculations import calculate_snr_from_reflectivity
 
+try:
```

Comment:
I think the main concern was ndimage which I do not know if it automatically ships with scipy


Dr Scott Collis
Purveyor of lofty observations (Atmospheric Scientist) 
Environmental Science Division
Argonne National Laboratory
Mb: +1 630 235 8025
Of: +1 630 252 0550
Become a Py-ART user today! http://arm-doe.github.io/pyart/ <http://arm-doe.github.io/pyart/>
> On Aug 9, 2016, at 10:39 AM, Jonathan J. Helmus <notifications@github.com> wrote:
> 
> In pyart/retrieve/__init__.py <https://github.com/ARM-DOE/pyart/pull/565#discussion_r74083767>:
> 
> > @@ -27,4 +27,10 @@
> >  from .gate_id import map_profile_to_gates, fetch_radar_time_profile
> >  from .simple_moment_calculations import calculate_snr_from_reflectivity
> >  
> > +try:
> Why does this need to be optionally imported? Everything in the module is a required dependency
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/ARM-DOE/pyart/pull/565/files/6a1724c8fca6d2a7f7c7175ceb51684ba82a0b95#r74083767>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAyYBwt-TtL9q_fhUbMjsYJrBEHWW6O3ks5qeJ8agaJpZM4JgLvz>.
>